src/test/java/com/bankofspring/hello/HelloWorldTest.java

```diff
@@ -0,0 +1,33 @@
+package com.bankofspring.hello;
+
+import static org.junit.Assert.*;
+
+import javax.naming.Context;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class HelloWorldTest {
+
+	private ApplicationContext appContext;
+	
+	@Before
+	public void setup(){
+		appContext = new ClassPathXmlApplicationContext("Spring-Module.xml");
+	}
+	
+	@Test
+	public void testHelloWorld() {
+		
+		HelloWorldBean helloWorld = (HelloWorldBean)appContext.getBean("helloBean");
+		
+		assertTrue("I am the Spring Framework - Yay!!!".equals(helloWorld.getName()));
```

Comment:
Giving this the optional first argument would give you a better error message if it fails. Also dxc suggested to me the other week that assertEquals is better than assertTrue as it gives better error messages (or assertThat( x, is(condition))).